requirements/base.in

```diff
@@ -3,7 +3,6 @@
 -c constraints.txt
 
 django
-pymongo>=2.7.2,<4.0.0
```

Comment:
This has no usage in this package. It was appeared via cookie cutter when this repo was created.